src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,295 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+
+public class Duke {
+    static final int MAX_TASKS = 100;
+    static final String EXCEPTION_1 = "Empty Description";
+    static final String EXCEPTION_2 = "Other Command";
+    static final String EXCEPTION_3 = "Mark/Unmark Out of Bounds";
+    static final String EXCEPTION_4 = "File Not Found";
+
+    public static ArrayList<Task> tasks = new ArrayList<>();
+    public static int numberOfTasks = 0;
+
+    public static void printDivider() {
+        System.out.println("\t----------------------------------------------------");
+    }
+
+    public static void printGreeting() {
+        printDivider();
+        System.out.println("\tHello! I'm Albot, your new best friend! ᕙ(`▿´)ᕗ");
+        System.out.println("\tWhat can I do for you today, friend? ʕ•́ᴥ•̀ʔっ♡");
+        System.out.println();
+        printDivider();
+    }
+
+    public static void printExitMessage() {
+        printDivider();
+        System.out.println("\tAww, you're leaving already? I'm gonna miss you (╥﹏╥)");
+        System.out.println("\tBut anyways bye! Hope to see you again soon, friend! (ɔ◔‿◔)ɔ ♥");
+        System.out.println();
+        printDivider();
+    }
+
+    public static void printExceptionMessage(String exceptionType) {
+        printDivider();
+
+        switch (exceptionType) {
+        case EXCEPTION_1 :
```

Comment:
Naming of the Exception constants could be improved. Right now the meaning would not be clear without the comments. Try to rename them like EMPTY_DESCRIPTION_EXCEPTION so that the meaning is always clear without needing to write comments at every usage.

Code Quality Guideline: Name Well